journal/journal.go

```
@@ -98,12 +104,7 @@ func Send(message string, priority Priority, vars map[string]string) error {
 
 		rights := syscall.UnixRights(int(file.Fd()))
 
-		/* this connection should always be a UnixConn, but better safe than sorry */
-		unixConn, ok := conn.(*net.UnixConn)
-		if !ok {
-			return journalError("can't send file through non-Unix connection")
-		}
-		unixConn.WriteMsgUnix([]byte{}, rights, nil)
+		conn.WriteMsgUnix([]byte{}, rights, socketAddr)
```

Maybe don't ignore the error here?